Phonebook.Frontend/src/styles.scss

```diff
@@ -137,6 +137,10 @@ app-root {
   min-height: 100%;
 }
 
+.pb-opacity-background {

```

Comment:
This is only specific to the navigation module so please put it into the stylesheet there. 
Further, maybe another button is something for you here: https://material.angular.io/components/button/examples
Maybe a stroked one?